### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by XinyuHou**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/XinyuHou/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
```
Comment (XinyuHou) [inline ReadMe.md]: why

Comment (XinyuHou) [inline ReadMe.md]: because !fix

Comment (XinyuHou) [inline ReadMe.md]: fixed !fixed

**Pull request merged**